### PR TITLE
BMCP-10 : Create Footer

### PR DESCRIPTION
**Pull request by 11vartika**
**Create Footer**

       Added  -
  1.  
WhatsApp Image 2021-06-29 at 7 22 58 PM
Screenshot from 2021-06-29 19-20-05
src / components/Footer/index.tsx
  2 . src / components/Footer/Footer.styles.css 

     Modified -
1.    App.tsx
 2.   routes.tsx

### CODE REVIEW

File: src/routes.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import { Footer } from "./components/Footer/Footer";
+import { Footer } from "./components/Footer";
```
Comment (nik-1236789) [inline src/routes.tsx:3]: We don't want it in the router.This needs to be displayed irrespective of the route.

**Review by nik-1236789 — CHANGES_REQUESTED** [review]
Looks good. Just get the commented query sorted!

File: src/components/Topbar/Topbar.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 export const Topbar: React.FC = () => {
     return (
         <div>
-            <h1>Topbar</h1>
+
```
Comment (2SkeePeR2) [inline src/components/Topbar/Topbar.tsx:6]: Remove the file if not required

Comment (2SkeePeR2) [inline src/components/Topbar/Topbar.tsx:6]: What is the purpose of this file? If not required, then delete it.

**Review by 2SkeePeR2 — CHANGES_REQUESTED** [review]

File: src/components/Topbar/Topbar.tsx (hunk 2 above)
Comment (11vartika) [reply src/components/Topbar/Topbar.tsx:6]: okay

Comment (nikhilsharma0025) [reply src/components/Topbar/Topbar.tsx:6]: just take the latest pull this will be fixed,then push that

Comment (11vartika) [reply src/components/Topbar/Topbar.tsx:6]: okay

**Review by 2SkeePeR2 — APPROVED** [review]

**Comment by truelegion47** [comment]
@nik-1236789 @2SkeePeR2 Is this ready to go ?

**Comment by 2SkeePeR2** [comment]
> @nik-1236789 @2SkeePeR2 Is this ready to go ? 

Approved by me.... ASK @nik-1236789

**Review by nik-1236789 — COMMENTED** [review]
@nikhilsharma0025 @rccoding discuss with each other and replicate this work.

**Comment by nikhilsharma0025** [comment]
I will work on this thing

**Pull request closed without merging**